x-pack/plugins/observability_solution/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.test.ts

```diff
@@ -2416,7 +2438,7 @@ describe('The metric threshold rule type', () => {
                         value: groupByKeys[key],
                       })),
                     }
-                : {}),
+                : { [ALERT_GROUP]: undefined }),
```

Comment:
The non groupByKeys case no longer is excluded from the object, also returning `['kibana.alert.group']: undefined`

Reply:
q: was there anything changed in this PR that caused this? or is this just cleaning up a case that's no longer valid?

Reply:
same reason as comment above - if there is no group-by in rule, we don't write `kibana.alert.group` in alert document, so I would keep it as `{}`